ballerina/Module.md

```diff
@@ -2,66 +2,119 @@
+
+Import the relevent Candid module into your Ballerina project.
+
+#### Charity Check PDF API
 ```ballerina
 import ballerinax/candid.charitycheckpdf;
+```
 
-charitycheckpdf:ApiKeysConfig apiKeyConfig = {subscriptionKey: "ENTER-THE-SUBSCRIPTION-KEY"};
-charitycheckpdf:Client charitycheckpdf = check new charitycheckpdf:Client(apiKeyConfig);
+#### Essentials API
+```ballerina
+import ballerinax/candid.essentials;
+```
 
-function getCharityCheckPDF() returns http:Response|error {
-    http:Response|error result = charitycheckpdf->/v1/pdf/["EMP-ID-NUM"];
-    return result;
-}
+#### Premier API
+```ballerina
+import ballerinax/candid.premier;
 ```
 
-### Essentials API
+### Step 2: Instantiate a new connector
 
-The following Ballerina program finds nonprofits using search criterias and explore essential information.
+Create an instance of `ApiKeysConfig` with the obtained Subscription Key and initialize the connector with it.
 
+#### Charity Check PDF API
 ```ballerina
-import ballerinax/candid.essentials;
+charitycheckpdf:ApiKeysConfig apiKeyConfig = {
+    subscriptionKey: "ENTER-THE-SUBSCRIPTION-KEY"
+};
+charitycheckpdf:Client charitycheckpdf = check new (apiKeyConfig);
+```
 
-essentials:ApiKeysConfig apiKeyConfig = {subscriptionKey: "ENTER-THE-SUBSCRIPTION-KEY"};
-essentials:Client essentials = check new essentials:Client(apiKeyConfig);
+#### Essentials API
+```ballerina
+essentials:ApiKeysConfig apiKeyConfig = {
+    subscriptionKey: "ENTER-THE-SUBSCRIPTION-KEY"
+};
+essentials:Client essentials = check new (apiKeyConfig);
+```
 
-function getV3Essesntials() returns essentials:V3EssentialsResponse|error {
-    essentials:V3Query query = {
-        search_terms: "candid"
-    };
-    essentials:V3EssentialsResponse|error result = essentials->/v3.post(query);
+#### Premier API
+```ballerina
+premier:ApiKeysConfig apiKeyConfig = {
+    subscriptionKey: "ENTER-THE-SUBSCRIPTION-KEY"
+};
+premier:Client premier = check new (apiKeyConfig);
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+
+#### Charity Check PDF API
+The following Ballerina program generates a PDF report to validate nonprofit status and eligibility with a 100% IRS-compliant charity check.
+
+```ballerina
+http:Response|error result = charitycheckpdf->/v1/pdf/["EMP-ID-NUM"];
     return result;
-}
 ```
 
-### Premier API
+#### Essentials API
+The following Ballerina program finds nonprofits using search criterias and explore essential information.
 
+```ballerina
+essentials:V3Query query = {
+    search_terms: "candid"
+};
+essentials:V3EssentialsResponse|error result = essentials->/v3.post(query);
+```
+
+#### Premier API
 The following Ballerina program retrieves data on a nonprofit's financials, people, DEI, and IRS compliance validation for the given employer id number.
 
 ```ballerina
-import ballerinax/candid.premier;
+premier:V3PublicProfile|error result = check premier->/v3/["EMP-ID-NUM"];
+```
```

Comment:
We need to mention executing the `bal run` command as the last step of the quickstart guide.